JakinChan200/site
# Updated metadata & Github logo

## Overview
 * Updated Metadata
 * Updated Github Logo to be the official one
 * Added boxes to the experience section

### To Do
* N/A

## Materials
 * N/A

## Manual Testing Guide
- Navigate to `site` folder
- Run `npm i`
- Run `npm run dev`

Pull requests should receive approval before merge.
